manim/mobject/geometry.py

```diff
@@ -838,6 +838,14 @@ def get_unit_vector(self):
     def get_angle(self):
         return angle_of_vector(self.get_vector())
 
+    def get_projection(self, point):
+        """
+        Return projection of a point onto the line
```

Comment:
Added punctuation. Also, a test would be nice -- either as a doctest (like in the suggestion), or as a separate (non-graphical) unit test in `tests`.

```suggestion
        """Return the projection of a point onto the line.

        Examples
        --------

        ::

            >>> import numpy as np
            >>> line = Line(LEFT, RIGHT)
            >>> line.get_projection(np.array([0, 1, 0]))
            array([0, 0, 0])

```